specifications/sep-payment-channel-mechanism.md

```diff
@@ -0,0 +1,657 @@
+    signers to V.
+    - One or more `SET_OPTIONS` operations adding R's signers to E.
+    - One `END_SPONSORING_FUTURE_RESERVES` operation that stops R sponsoring
+    future reserves of subsequent operations.
+  - One or more `PAYMENT` operations depositing I's contribution to E.
+  - One or more `PAYMENT` operations depositing R's contribution to E.
+  - One or more `PAYMENT` operations depositing R's reserves to V, for each
+  trustline on E that will be used to sponsor claimable balances at
+  disbursement.
+  
+- C_i, see [Update](#Update) process.
+
+- D_i, see [Update](#Update) process.
+
+#### Update
+
+Participants use the update process to agree, and re-agree, on a new final state
+for the channel. Participants will agree on a new final state when making
+payments to one another within the channel.
+
+For example, if the channel initial state is $30 of which $10 belongs to I and
+$20 belongs to R, the first closing transaction will disburse $10 to I and $20
+to R. If I makes a payment of $2 to R, this update process will involve I and R
+agreeing on a new declaration and aclosing transaction that supersedes all
+previous declaration and closing transaction and that will disburse $8 to I and
+$22 to R.
+
+To update the payment channel state, the participants:
+
+1. Increment i.
+2. Sign and exchange a closing transaction C_i.
+3. Sign and exchange a declaration transaction D_i.
+
+The transactions are constructed as follows:
+
+- C_i, the _closing transaction_, disburses funds to R and changes the signing
+weights on E such that I unilaterally controls E.  C_i has source account E,
+sequence number s_i+1, a `minSeqAge` of O (the observation period time
+duration), and a `minSeqLedgerGap` of O (the observation period ledger count).
+
+  The `minSeqAge` and `minSeqLedgerGap` prevents a misbehaving party from
+  executing C_i when the channel state has already progressed to a later
+  iteration number, as the other party has the period O to invalidate C_i by
+  submitting D_i' for some i' > i.
+  
+  C_i contains operations:
+  - One `BEGIN_SPONSORING_FUTURE_RESERVES` operation that specifies reserve
+  account V as a sponsor of future reserves.
+  - One `CREATE_CLAIMABLE_BALANCE` operation for each trustline that is
+  disbursing funds to R.
+  - One `END_SPONSORING_FUTURE_RESERVES` operation that confirms reserve
+  account V's sponsorship.
+  - One or more `SET_OPTIONS` operation adjusting escrow account E's
+  thresholds to give I full control of E, and removing R's signers.
+  - One or more `SET_OPTIONS` operation adjusting reserve account E's
+  thresholds to give R full control of V, and removing I's signers.
+
+- D_i, the _declaration transaction_, declares an intent to execute the
+corresponding closing transaction C_i.  D_i has source account E, sequence
+number s_i, and `minSeqNum` set to s_e.  Hence, D_i can execute at any time, so
+long as E's sequence number n satisfies s_e <= n < s_i.  Because C_i has source
```

Comment:
❓ going off my s_e question, could't this just be
```suggestion
long as E's sequence number n satisfies s <= n < s_i.  Because C_i has source
```

Reply:
When e is zero, then s_e has the same value as s, but when e is greater than zero, it will have a greater value.